public/index.html

```diff
@@ -5,18 +5,9 @@
     <meta charset = "UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet">
   </head>
-
   <body>
-
-
-
-
     <div id="reviewsSection"></div>
-
-    <script src="./bundle.js"></script>
+    <script src="./reviews-bundle.js"></script>
```

Comment:
Yay, let's fix all these file naming issues early!

Reply:
I was inspired by you!